lib/version_utils.pm

```diff
@@ -299,6 +300,21 @@ sub is_server {
     return check_var('SLE_PRODUCT', 'sles');
 }
 
+=head2 get_distro
+
+    get_distro();
+
+Return distribution string (SLE, Tumbleweed, Leap)
+TODO: Add other distributions (i.e. caasp, jeos, ...)
+=cut
+sub get_distro {
+    return 'SLE' if (is_sle);
+    if (is_opensuse) {
+        return 'Tumbleweed' if (is_tumbleweed);
+        return 'Leap';
+    }
```

Comment:
@foursixnine closing as `get_distro` probably not needed. Maybe `get_product_version` could be used.

NOTE: name `get_distro()` is probably misleading. Whole thing was needed due 
* sles: `DISTRI=sle`, `VERSION=15-SP1`
* openSUSE Tumbleweed: `DISTRI=opensuse`, `VERSION=Tumbleweed`.
The need of this function for me was the fact, that we use different set of variables when generating repositories (`DISTRI` for SLES, but `VERSION`)

See 
https://download.opensuse.org/repositories/devel:/openSUSE:/QA:/Leap:/15/openSUSE_Leap_15.1/
https://download.opensuse.org/repositories/devel:/openSUSE:/QA:/Tumbleweed/openSUSE_Tumbleweed/

vs. 
http://download.suse.de/ibs/QA:/Head/openSUSE_Tumbleweed/
http://download.suse.de/ibs/QA:/Head/SLE-15-SP1/

=> OBS and IBS download URL reflect the difference. Unfortunately openSUSE needs to use OBS (cannot reach IBS) and SLES IBS (even there are some packages in OBS, usually not for all SLES versions or archs).